moveit_msgs/PlanningSceneWorld world
moveit_msgs/RobotState robot
